src/appleseedmaya/shadingnodemetadata.cpp

```diff
@@ -125,6 +135,11 @@ OSLParamInfo::OSLParamInfo(const asf::Dictionary& paramInfo)
                 defaultValue.push_back(paramInfo.get<float>("default"));
                 hasDefault = true;
             }
+            else if (paramType == "float[2]")
+            {
+                getFloat2Default(paramInfo, defaultValue);
```

Comment:
We can inline the code of getFloat2Default here If the function is not reused anywhere.
It makes the code more readable.